basic-assembly/basic-rules/pom.xml

```diff
@@ -15,19 +15,19 @@
     <modelVersion>4.0.0</modelVersion>
 
     <parent>
-        <artifactId>antenna-basic-assembly</artifactId>
         <groupId>org.eclipse.sw360.antenna</groupId>
+        <artifactId>basic-assembly</artifactId>
         <version>${revision}${qualifier}</version>
     </parent>
 
-    <artifactId>antenna-basic-rules</artifactId>
+    <artifactId>basic-rules</artifactId>
     <packaging>jar</packaging>
```

Comment:
I prefer common rules not basic rules, the rules are commonly applicable and do not express a legal interpretation, but some structural requirements on a component